src/librustdoc/html/static/main.js

```diff
@@ -385,6 +386,127 @@
                 }
             }
 
+            function transformResults(results, isType) {
+                var out = [];
+                for (i = 0; i < results.length; ++i) {
+                    if (results[i].id > -1) {
+                        var added = false;
```

Comment:
This line seems like it was left behind by accident.

Reply:
Good catch!